articles/client-platforms/angular2/08-calling-apis.md

```diff
@@ -7,13 +7,13 @@ description: This tutorial will show you how to use angular2-jwt library in Angu
   link: 'https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/08-Calling-Api',
 }) %>_
 
-The reason for implementing authentication in the first place is to protect information. In this case your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. Auth0 suggests you conform to RFC standard by sending the token through Authorization header.
+Auth0 exposes an assortment of API endpoints to assist you with authentication in your application. Auth0 suggests you conform to the RFC standard by sending the token through Authorization header when calling an API.
 
 ## Meet AuthHttp
```

Comment:
We should change this heading. Let's call it `Sending Authenticated HTTP Requests`